Plugin~/WebRTCPlugin/WebRTCPlugin.vcxproj

```diff
@@ -74,16 +74,24 @@
     <LinkIncremental>false</LinkIncremental>
     <TargetName>webrtc</TargetName>
     <OutDir>..\..\Runtime\Plugins\x86_64</OutDir>
+    <IncludePath>C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.1\include;C:\VulkanSDK\1.1.121.2\Include;$(IncludePath)</IncludePath>
```

Comment:
We need to use `environment variables` to support build plugin on another drive.
Can we use `$CUDA_PATH`?

Reply:
ok

Reply:
Should we also use `$VULKAN_PATH`

Reply:
For Cuda, Let's use `CUDA_INCLUDE_PATH` and `CUDA_LIB_PATH`